images/installer/Dockerfile.upi.ci

```diff
@@ -58,9 +58,9 @@ RUN curl -sSL "${ALIYUN_URI}" --output /tmp/aliyun-cli-linux-latest-amd64.tgz &&
 RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
 RUN python get-pip.py 'pip<21.0'
 RUN python -m pip install pyopenssl
-ENV CLOUDSDK_PYTHON=/usr/bin/python
+ENV CLOUDSDK_PYTHON=/usr/bin/python3
 
-RUN python3 -m pip install yq
+RUN python3 -m pip install yq pyopenssl
```

Comment:
Depending on the version of the gcloud cli, we need to install `cryptography` instead. See https://github.com/openshift/installer/commit/8eb6600b2d730a4f5b612221d2aa2729b6806a52

Reply:
Is there a reason to keep python2 at all here or can we bump everything to python3?

Reply:
python3 is the default in rhel8. I wonder if we can drop `images/installer/Dockerfile.upi.ci` and use `images/installer/Dockerfile.upi.ci.rhel8` instead.
@patrickdillon do you know why that's not the case?

Reply:
> Depending on the version of the gcloud cli, we need to install `cryptography` instead. See [8eb6600](https://github.com/openshift/installer/commit/8eb6600b2d730a4f5b612221d2aa2729b6806a52)

After upgrading pip3, `python3 -m pip install pyOpenSSL` can succeed, see the passed [job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7118/pull-ci-openshift-installer-master-e2e-gcp-ovn-upi/1650474432583438336) with the latest commit, thanks!